phpBB/phpbb/notification/type/report_pm.php

```diff
@@ -84,9 +84,8 @@ static public function get_item_parent_id($pm)
 	*/
 	public function is_available()
 	{
-		$m_approve = $this->auth->acl_getf($this->permission, true);
-
-		return (!empty($m_approve));
+		return !empty($this->auth->acl_get($this->permission)) &&
```

Comment:
Please move the check for `$this->config['allow_pm_report']` to before the call to `acl_get()`.